### PR TITLE
Add Mural Card component [MI-758]

### PR DESCRIPTION
**Pull request by btuohy-mural**
This PR refactors out the MuralCard from the MuralPicker component. 

You can check out the new component by running Storybook.
```
cd storybook
npm run storybook
```

This PR does NOT update the MuralPicker to use the new MuralCard component. There is another dev who will need to use this new component, so I wanted to get it out there ASAP and then update the MuralPicker in another PR. It should be quick though.

Please help me make sure the package stuff is right. That is something I struggle with, and I copied the package file from a different project. 😳 

Here are some screenshots of what it looks like in Storybook. 💅  It's hard to tell in the screenshot, but when the card is selected it moves ever-so-slightly upward, so there is a neat movement effect.

image
image
image
image
image
image
image
image
image
image

### CODE REVIEW

File: packages/mural-card/src/components/mural-card/index.tsx [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+export type CardItemSource = {
+  title: string;
+  thumbnailUrl: URL | string;
+  details?: string;
+  initials?: string;
+};
+
+export interface MuralCardPropTypes {
+  source: CardItemSource;
+  onClick: EventHandler;
```
Comment (KavenTheriault) [inline packages/mural-card/src/components/mural-card/index.tsx:17]: Maybe this `onClick` could be optional? I can think of a use case in Teams where we would like to use this component without a click event.

File: packages/mural-card/src/components/mural-card/index.tsx [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+
+    font-size: ${props => props.size.fontSize};
+    font-family: 'Proxima Nova', -apple-system, BlinkMacSystemFont, 'Segoe UI',
+      'Roboto', 'Oxygen', 'Ubuntu', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+      sans-serif;
+    color: ${({ theme }) => theme.primaryTextColor};
+
+    margin: 0.675em 0.375em;
+    border: 1px solid
+      ${props => (props.isSelected ? '#ff0066' : 'rgba(0, 0, 0, 0.12)')};
```
Comment (KavenTheriault) [inline packages/mural-card/src/components/mural-card/index.tsx:81]: Can you update this color to the new Mural red? (`#E02935`)

File: packages/mural-card/src/components/mural-card/index.tsx [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+
+export interface MuralCardPropTypes {
+  source: CardItemSource;
+  onClick: EventHandler;
+  cardSize?: CardSize;
+  isSelected?: boolean;
+  theme?: 'light' | 'dark';
+}
+
+export const FONT_FAMILY = 'Proxima Nova, sans-serif';
```
Comment (KavenTheriault) [inline packages/mural-card/src/components/mural-card/index.tsx:23]: Those 3 variables are never used.

**Comment by btuohy-mural** [comment]
Closed in favor of https://github.com/MuralDevKit/mural-integrations-sdk/pull/87

**Pull request closed without merging**